### PR TITLE
AB#65906 Added Funding page

### PR DESCRIPTION
**Pull request by Antek012**
image

### CODE REVIEW

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.InvestmentLoans.BusinessLogic/Organization/NumericValidator.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        _value = value;
+        _fieldName = fieldName;
+        _operationResult = operationResult;
+    }
+
+    public static implicit operator decimal(NumericValidator v) => v._value ?? 0;
+
+    public static implicit operator int(NumericValidator v) => Convert.ToInt32(v._value ?? 0);
+
+    public static NumericValidator For(decimal? input, string fieldName, OperationResult? operationResult = null)
```
Comment (jjankowski87) [inline src/HE.InvestmentLoans.BusinessLogic/Organization/NumericValidator.cs:23]: add display name for field for more user friendly error message

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investment.AHP.Contract/Scheme/Scheme.cs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+namespace HE.Investment.AHP.Contract.Scheme;
+
+public record Scheme(string SchemeId, decimal? RequiredFunding, int? HousesToDeliver);
```
Comment (Carq) [inline src/HE.Investment.AHP.Contract/Scheme/Scheme.cs:3]: `Schema` -> `SchemaModel`?

**Review by Carq — APPROVED** [review]

File: src/HE.InvestmentLoans.BusinessLogic/Organization/NumericValidator.cs (hunk 1 above)
Comment (Antek012) [reply src/HE.InvestmentLoans.BusinessLogic/Organization/NumericValidator.cs:23]: I will consider this.

File: src/HE.Investment.AHP.Contract/Scheme/Scheme.cs (hunk 2 above)
Comment (Antek012) [reply src/HE.Investment.AHP.Contract/Scheme/Scheme.cs:3]: I will consider this.

**Pull request merged**